previous-versions/ZAM_fnc_EZMLiteV2.1.1.sqf

```diff
@@ -5979,6 +5980,274 @@ MAZ_EZM_fnc_initFunction = {
+				"colorCorrections" ppEffectCommit 0;
+				"colorCorrections" ppEffectEnable true;
+
+			};
+		};
+
+		if (_cinematicType == "Flyby") then {
+			comment "in flyby mode, we select the module location as our target, and the camera paths are automatically designated at 0 and 90 degrees";
+			private _camTarget = "Land_HelipadEmpty_F" createVehicleLocal _target;
+			private _circleRadius = 200;
+			private _camHeight = 200;
+			private _camSrc0 = [_target select 0, (_target select 1) + _circleRadius, (_target select 2) + _camHeight];
+			private _camSrc90 = [(_target select 0) + _circleRadius, _target select 1, (_target select 2) + _camHeight];
+			
+			private _camera = "camera" camCreate _camSrc0;
+			_camera cameraEffect ["internal", "back"];
+			_camera camPrepareTarget _camTarget;
+			_camera camSetFov 1;
+			_camera camCommitPrepared 0;
+
+			_camera camPreparePos _camSrc90;
+			_camera camCommitPrepared 15;
+			waitUntil { camCommitted _camera };
+			cutRsc ["RscStatic", "PLAIN"];
+			sleep 0.4;
+			_camera cameraEffect ["terminate", "back"];
+			cutText ["", "BLACK IN", 2];
+			[1, 2, true, true] call BIS_fnc_cinemaBorder;
+
+		};
+
+		comment "clean up scripts";
+		if(_zeusCanSeeCutscene) then {
+			[] call MAZ_EZM_fnc_refreshInterface;
+		};
+
+		"colorCorrections" ppEffectAdjust[1,1,0,[0,0,0,0],[1,1,1,1],[0,0,0,0]];
+		"colorCorrections" ppEffectCommit 0;
+		"colorCorrections" ppEffectEnable true;
+
+		{
+			if !(_x == vehicle _x) then {
+				[vehicle _x, true] remoteExec ["enableSimulationGlobal", 2];
+			};
+		} forEach allPlayers;
+
+		["Intro cinematic completed."] call MAZ_EZM_fnc_systemMessage;
+
+	};
+
+	HYPER_EZM_fnc_introCinematicModule = {
+		params ["_entity"];
+		private _dialogTitle = "Intro Cinematic";
+		private _content = [
+			[
+				"COMBO",
+				"Cinematic Type",
+				[
+					["Flyby"],
+					["Flyby"],
+					0
+				]
+			],
+			[
+				"COMBO",
+				"Background Song",
+				[
+					["random", "epic", "action", "stealth"],
+					["Random Event Track", "Epic", "Action", "Stealth"],
+					0
+				]
+			],
+			[
+				"EDIT",
+				"Intertitle 1",
+				[
+					"",
+					1
+				]
+			],
+			[
+				"EDIT",
+				"Intertitle 2",
+				[
+					"",
+					1
+				]
+			],
+			[
+				"TOOLBOX:YESNO",
+				["Zeus Can See Cutscene?","Enabling this may break the camera for Zeus players."],
+				[false]
+			],
+							[
+				"COMBO",
+				"Post-Process Filter",
+				[
+					["none", "highcontrast", "blue", "dull", "yellowgamma", "greengamma"],
+					["None", "High Contrast", "Blue", "Dull", "Yellow Gamma", "Green Gamma"],
+					0
+				]
+			]
+		];
+		private _onConfirm = {
+			params ["_values", "_args", "_display"];
+			_values params ["_cinematicType","_backgroundSong","_text1","_text2","_zeusCanSeeCutscene","_postProcess"];
+			private _intertitles = [_text1,_text2];
+			private _target = [true] call MAZ_EZM_fnc_getScreenPosition;
```

Comment:
`[true] call MAZ_EZM_fnc_getScreenPosition` should be passed as an argument to `MAZ_EZM_fnc_createDialog`. 

As it is now, it gets the mouse position only when the Zeus presses OK, leading to the cinematic being focused on an unexpected position different from where the module was placed.